tests/bindings/test_python.sh

```diff
@@ -3,12 +3,13 @@
 # Author:
 #   Martin Preisler <mpreisle@redhat.com>
 
-. ../test_common.sh
+. $builddir/tests/test_common.sh
+set -x
```

Comment:
!

Reply:
@mpreisler Nice catch! Thank you. I have forgotten this while debugging and I have committed it by accident. I'll remove it.